aiojobs/_job.py

```diff
@@ -50,6 +56,10 @@ def pending(self) -> bool:
     def closed(self) -> bool:
         return self._closed
 
+    @property
+    def name(self) -> str:
```

Comment:
```suggestion
    def get_name(self) -> str:
```

My gut feeling is that we should copy the interface from asyncio.Task.

Reply:
Sounds good to me. I had it get_name originally and then thought it looked strange with the property syntax for closed, pending and running.